web/util/date.ts

```diff
@@ -2,24 +2,20 @@ import { formatInTimeZone } from 'date-fns-tz';
 import { addMinutes, intervalToDuration } from 'date-fns';
 import enUS from 'date-fns/locale/en-US';
 
-export const formatTime = (date: Date, timezone: string) =>
-  formatInTimeZone(date, timezone, 'HH:mm', { locale: enUS });
-
-export const formatTimeRange = (
-  date: Date,
-  duration: number,
-  timezone: string
-) =>
-  `${formatTime(date, timezone)}–${formatTime(
-    addMinutes(date, duration),
-    timezone
-  )}`;
+export const formatTime = (date: Date, timezone: string, meridiem?: boolean) =>
+  formatInTimeZone(date, timezone, `HH:mm${meridiem ? ' aa' : ''}`, {
```

Comment:
This is mixing 24-hour format with AM/PM marker.

E.g. `formatTime(parseISO('2022-04-05T11:59Z'), 'Europe/Oslo', true)` shows "13:00 PM". Expected: "1:00 PM".

```suggestion
  formatInTimeZone(date, timezone, meridiem ? 'h:mm aa' : 'HH:mm', {
```